pkgs/applications/audio/stochas/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, pkg-config, libX11, libXrandr, libXinerama, libXext, libXcursor, freetype, alsaLib, libjack2 }:
+
+stdenv.mkDerivation rec {
+  pname = "stochas";
+  version = "1.3.4";
+
+  src = fetchFromGitHub {
+    owner = "surge-synthesizer";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0b26mbj727dnygavz4kihnhmnnvwsr9l145w6kydq7bd7nwiw7lq";
+    leaveDotGit = true; # for SURGE_VERSION
```

Comment:
Can we set this with ``cmake -B build -DSTOCHAS_VERSION=xxx``? https://github.com/surge-synthesizer/stochas/blob/main/CMakeLists.txt#L6
``

Reply:
Please elaborate, I'm not following.

Reply:
It is better for reproducibility to remove the .git directory. We can set the version maybe with this flag or through patching.

Reply:
Looks like I made a mistake.
When I remove ``leaveDotGit = true;`` and change the hash, I get back the original hash from the build.
When I change it back, there is no build and the original pkg get's installed.